daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -15,19 +15,33 @@ import (
 	"os"
 	"path"
 	"path/filepath"
+	"strings"
 	"syscall"
 	"unsafe"
 
 	"github.com/docker/docker/daemon/graphdriver"
 	"github.com/docker/docker/pkg/idtools"
 	"github.com/docker/docker/pkg/mount"
+	"github.com/docker/docker/pkg/parsers"
+	"github.com/docker/go-units"
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
 func init() {
 	graphdriver.Register("btrfs", Init)
 }
 
+var (
+	defaultMinFreeSpace uint64 = 10 * 1024 * 1024 * 1024
```

Comment:
Where is this number coming from? From the code it looks more like a limit than a default.